src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -352,6 +352,9 @@ private boolean performForApplication(Run<?, ?> build, FilePath workspace, EnvVa
 
                 int appIndex = applications.indexOf(application);
 
+                logger.println("HOCKEYAPP_INSTALL_URL" + installUrl);
```

Comment:
The catch with putting this outside the `if` block on line 316, is that even though the value is available it might not be set in the environment data.  Would this cause confusion?

Reply:
thank you for reviewing!!! I think you meant line 361 not 316  ? ie this : https://github.com/AnneTheAgile/hockeyapp-plugin/blob/f55ab778243a04de69d51fefcbfdab8d802db392/src/main/java/hockeyapp/HockeyappRecorder.java#L361
I don't see that envData is ever set at all. Am I missing something? 
My preference is regardless of what future code wants, ie environment variable usages, the log should indicate what it did since it does create links in the html for the build. Those links are not available by scraping the console log, but that is what I would like to do.
https://github.com/ohoeltke/hockeyapp-plugin/search?q=envdata&type=Code&utf8=%E2%9C%93